### PR TITLE
Adding a Facebook provider

### PR DESCRIPTION
**Pull request by runxc1**
Updating to add a Facebook provider as well as include a way to request specific fields from Facebook

### CODE REVIEW

File: src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml [hunk 1]
```diff
@@ -278,6 +278,34 @@
...
+
+  <Provider Name="Facebook" Documentation="https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow">
+    <Environment Issuer="https://www.facebook.com/">
+      <Configuration AuthorizationEndpoint="https://www.facebook.com/v16.0/dialog/oauth"
+                     TokenEndpoint="https://graph.facebook.com/v16.0/oauth/access_token"
+                     UserinfoEndpoint="https://graph.facebook.com/v16.0/me">
+        <CodeChallengeMethod Value="S256" />
+      </Configuration>
+
+      <Scope Name="email" Default="true" Required="true" />
```
Comment (kevinchalet) [inline src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml]: Are we sure the `email` scope is really mandatory? Can you please try with this line removed?

**Review by kevinchalet — COMMENTED** [review]
Thanks for your PR!

File: src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml (hunk 1 above)
Comment (runxc1) [reply src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml]: Yeah it works if you remove that line,  also works if you change it from Required to false.

Comment (kevinchalet) [reply src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml]: Great, thanks!

**Comment by kevinchalet** [comment]
Merged. Congrats for your first contribution to OpenIddict! 🎉

**Pull request merged**